docs/generated/cli/create-nx-workspace.md

```diff
@@ -77,7 +77,7 @@ Workspace name (e.g. org name)
 
 Type: boolean
 
-Use Nx Cloud
+Setup distributed caching using Nx Cloud (It's free and doesn't require registration.)
```

Comment:
Hello, sorry to hop in, just browsing recent PRs out of a curiosity, but "set up" may be a better fit here? Usually I have heard it said that "setup" is a noun and "set up" is a verb.

```suggestion
Set up distributed caching using Nx Cloud (It's free and doesn't require registration.)
```

Reply:
Yeah, it has to be set up cause it's a verb